Miscellaneous/scf-record.py

```diff
@@ -0,0 +1,2 @@
+{'EthaneA 1Cx2': (8, 7), 'PropaneA 1Cx3': (8, 8), 'PropaneA cc-c': (8, 8), 'ButaneA 1Cx4': (10, 10), 'ButaneA 2Cx2': (10, 10), 'ButaneA ccc-c': (10, 10), 
+ 'HexaneA 1Cx6': (10, 11), 'Hexane 2Cx3': (10, 10), 'Hexane 3Cx3': (10, 10)}
```

Comment:
When I was noticed that the journal was taking 20+ minutes to run this notebook, I thought there was an issue. But running a marathon takes time.

It seems that in the case of hexane, the more fragments our guesses have, the more cycles are needed for convergence. If that were true, it would defeat the purpose of our approach, wouldn't it? 

Is it possible to run the molecule scf once, instead of running the mol scf every time a new guess is entered? Would that save us time?

Reply:
And of course, let me know what you think the next step should be. We could move up to larger alkane chains if you think that would clarify this.